### ISSUE TITLE
PHP __construct() issue

### ISSUE THREAD

**Issue by bhubbard** [idxbroker/wordpress-plugin#1]
There are several notices and warnings caused by the IDX Broker plugin that need to be fixed.
```
Notice: The called constructor method for WP_Widget is deprecated since version 4.3.0! Use
__construct()
instead.

Notice: The called constructor method for WP_Widget is deprecated since version 4.3.0! Use
__construct()
instead. 

Warning: Cannot modify header information - headers already sent 

Warning: Cannot modify header information - headers already sent 
```

**Comment by bhubbard** [comment]
Looks like this bug occurs twice, here are the two offending pieces of code:

https://github.com/idxbroker/wordpress-plugin/blob/master/idx-broker-platinum/idx-broker-widgets.php#L109-L120

https://github.com/idxbroker/wordpress-plugin/blob/master/idx-broker-platinum/omnibar/idx-omnibar-widget.php#L46-L53

**Comment by bhubbard** [comment]
This is fixed in #2
